### DISCUSSION TITLE
Define multiple parcels/packages per ProductVariation

### DISCUSSION THREAD

**Discussion by adevade** [lunarphp/lunar#1306] [Feature Requests]
At the moment, the measurements and volume are stored directly on each row of the `product_variants` table. A product can consist of multiple packages, think large furniture at IKEA.

It would be good if this was broken out into its own table, like `product_variant_packages` or similar, and made a OneToMany relationship instead.

What do you think?

**Comment by glennjacobs** [comment]
It's a fair amount of complexity that the majority won't use, so it would need some consideration to avoid a complex UI.

Also, it's likely to have a knock-on effect on shipping logic etc. 

In terms of naming, previously I've seen it called "boxes". 

I think I would want to wait until our Filament panel is available before moving onto these complexities.

**Reply by adevade** [reply]
Yeah, it's probably a feature for a small minority of users. But we could really use it at my job for example, because we ship very large products that has to be in multiple parcels.

Regarding the UI, I think it could probably look the same as it does now? Just with a button below to add more (optional) "rows" of boxes?

I haven't looked much at Lunars shipping logic yet, so no comment or suggestions there yet.

**Reply by glennjacobs** [reply]
This may want to be considered when we look at inventory.
